.github/workflows/node.js-tests.yml

```diff
@@ -130,7 +130,7 @@ jobs:
     strategy:
       matrix:
         node-version: [14.x]
-        locale: [chinese, espanol]
+        locale: [chinese, chinese-traditional, espanol, italian]
```

Comment:
To be reverted, the rest all looks good.

Reply:
Resolving as the change was reverted.